plaso/cli/helpers/elastic_output.py

```diff
@@ -18,7 +18,7 @@ class ElasticSearchServerArgumentsHelper(server_config.ServerArgumentsHelper):
   """Elastic Search server CLI arguments helper."""
 
   _DEFAULT_SERVER = '127.0.0.1'
-  _DEFAULT_PORT = 9200
+  _DEFAULT_PORT = None
```

Comment:
Please remove the port changes from this PR, to keep it focused on just the SSL certificate changes.

Reply:
If the port is set here the `_Connect` functions in `shared_elastic.py` will always use `elastic_hosts = [{'host': self._host, 'port': self._port}]`. This way ssl, subdirectories and other ports cannot be set. The default port in elasticsearch-py is 9200, so I does not need to be set here.